src/ghdcbot/adapters/storage/sqlite.py

```
@@ -174,17 +189,11 @@ def list_contribution_summaries(
         self,
         period_start: datetime,
         period_end: datetime,
-        weights: dict[str, int],
+        weights: dict[str, int] | None = None,
```

Ruff (0.15.18)

[warning] Unused method argument: `weights`

(ARG002)